scripts/convert_to_coreml.py

```
@@ -195,7 +207,13 @@ def save_package(model, suffix):
     ]
 
     for suffix, quantize in quantization_steps:
-        model_to_save = quantize(mlpackage_model)
+        try:
+            model_to_save = quantize(mlpackage_model)
+        except Exception as exc:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)